HealthcareSystem/Backend/Model/Exceptions/HealthClinicException.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Backend.Model.Exceptions
 {
-    class HealthClinicException : Exception
+    public class HealthClinicException : Exception
     {
         public HealthClinicException() : base() { }
         public HealthClinicException(String message) : base(message) { }
```

Comment:
This class is like "general" exception at the application level. This is the way for creating own exceptions which was explained by the professor and teaching assistants. In the case of name of this class, it was a suggestion your colleagues from DevOps, and I think the name fully corresponds to the purpose.

Reply:
I think it's definitely a descriptive name, since the responsibility of the class is just to act as the root of the exception hierarchy - thus, it's 'SystemName' + 'Exception'. Of course, you can't really tell when it's meant to be thrown from the name, because it's not meant to be thrown at all.

Reply:
Alright, then it is okay, I am wondering what the usage of this is, now I understand that is the root of the exception hierarchy. Thanks.